Source/WebKit/WebProcess/WebPage/WebPage.messages.in

```diff
@@ -819,4 +819,6 @@ GenerateSyntheticEditingCommand(enum:uint8_t WebKit::SyntheticEditingCommandType
     CloseCurrentTypingCommand()
 
     SimulateClickOverFirstMatchingTextInViewportWithUserInteraction(String targetText) -> (bool success)
+
+    SetPageScaleFactor(float scaleFactor, int x, int y) -> ()
```

Comment:
Why aren't we sending an `IntPoint` over IPC instead of `x` & `y` separately? Seems that what the Page needs eventually anyway.